src/Aspire.Dashboard/Components/Controls/PauseResumeButton.razor

```diff
@@ -0,0 +1,16 @@
+﻿@using Aspire.Dashboard.Resources
+@inject IStringLocalizer<ControlsStrings> Loc
+
+<FluentButton Appearance="Appearance.Lightweight"
+              Title="@(IsPaused ? Loc[nameof(ControlsStrings.ResumeButtonTitle)] : Loc[nameof(ControlsStrings.PauseButtonTitle)])"
```

Comment:
I like the tooltip. I noticed the remove button doesn't have a tooltip. Could you update it to have one ("Remove data") while you're here?